### PR TITLE
Initial code checks using PMD

### PR DESCRIPTION
**Pull request by cristianonicolai**
Included minimal set of rulesets: basic, imports and unused code.
Fixes for conforming with applied rules
Code formatting on changed files

### CODE REVIEW

**Comment by mswiderski** [comment]
can we postpone any bigger changes to console codebase I am in a middle of removing the jbpm runtime out of it and it affects "quite few" classes?

I hope to have something that runs today or tomorrow without jbpm runtime...

**Comment by jhrcek** [comment]
Wow, this is great. Now we'll finally be able to focus on relevant changes!  :+1:

File: jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/DeploymentDescriptorModel.java [hunk 1]
```diff
@@ -245,35 +245,20 @@ public boolean equals(Object o) {
     @Override
     public int hashCode() {
         int result = persistenceUnitName != null ? persistenceUnitName.hashCode() : 0;
-        result = ~~result;
```
Comment (jhrcek) [inline jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/DeploymentDescriptorModel.java]: Is it ok to delete these `result = ~~result;` ? According to this [stackoverflow answer](http://stackoverflow.com/questions/29736952/what-is-the-meaning-of-double-tilde-in-java) they might make a difference..

Comment (manstis) [inline jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/DeploymentDescriptorModel.java]: Nooooooo!

We started adding those operators as we found HashCode would behave really
strangely sometimes in GWT compiled code. On investigation GWT uses the ~~
operator extensively in its JRE emulated libraries' HashCode... so we
followed and our HashCode implementations behaved as expected.

Any class destined for the client side should, in my opinion use the ~~
operator in its HashCode implementation... or we'll return to having
"really hard to fix" bugs come back!

**Comment by manstis** [comment]
-1 (use of ~~ operator in client-side HashCode implementations *should* remain).

File: jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/DeploymentDescriptorModel.java (hunk 1 above)
Comment (cristianonicolai) [inline jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/DeploymentDescriptorModel.java]: No worries, I'll review this change to not change the `hashcode` method.

**Comment by cristianonicolai** [comment]
@mswiderski no worries delaying this change but I was expecting we would use some sort of feature branch for myself and Neus to work on the jbpm runtime removal before merging it to master.

**Comment by mswiderski** [comment]
it will be a feature branch but believe me, merging so much changes to feature branch where you'll have several commits diverged is not easy nor fun.

**Comment by cristianonicolai** [comment]
@mswiderski @jhrcek @manstis @nmirasch I've revisited this PR after the latest changes on master, please have another look.

**Comment by mswiderski** [comment]
shouldn't we remove the commented code in QuickNewTaskPopup?

and I agree with @manstis about the client hashCode, but that seems to be already covered. so  +1 from me

**Pull request merged**